### ISSUE TITLE
Crash on hentries.js

### ISSUE THREAD

**Issue by dg01d** [strugee/lazymention#45]
As discussed on #indieweb-dev:

```
assert.js:42
  throw new errors.AssertionError({
  ^
 
AssertionError [ERR_ASSERTION]: false == true
    at /usr/lib/node_modules/lazymention/lib/hentries.js:138:4
    at _handleNode (/usr/lib/node_modules/lazymention/lib/hentries.js:101:3)
    at handleNode (/usr/lib/node_modules/lazymention/lib/hentries.js:124:3)
    at /usr/lib/node_modules/lazymention/lib/app.js:96:6
    at Array.forEach (<anonymous>)
    at Arguments.<anonymous> (/usr/lib/node_modules/lazymention/lib/app.js:94:16)
    at Object.External.get (/usr/lib/node_modules/lazymention/node_modules/microformat-node/lib/index.js:4519:29)
    at /usr/lib/node_modules/lazymention/lib/app.js:87:8
    at /usr/lib/node_modules/lazymention/lib/get.js:58:5
    at ConcatStream.<anonymous> (/usr/lib/node_modules/lazymention/node_modules/concat-stream/index.js:36:43)
    at emitNone (events.js:120:20)
    at ConcatStream.emit (events.js:218:7)
    at finishMaybe (/usr/lib/node_modules/lazymention/node_modules/readable-stream/lib/_stream_writable.js:607:14)
    at afterWrite (/usr/lib/node_modules/lazymention/node_modules/readable-stream/lib/_stream_writable.js:470:3)
    at _combinedTickCallback (internal/process/next_tick.js:144:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

**Comment by strugee** [comment]
URL that triggers this crash: https://ascraeus.org/micro/1516031264/

**Comment by strugee** [comment]
Okay, so the problem here is that the submission is an individual page, not a feed.

I'm not sure what to do here. I guess probably if the page only includes one h-entry and no h-feed, we should just assume the page URL is canonical?

@dg01d in the meantime you can work around this by submitting your feed, for example https://ascraeus.org/micro/

**Comment by strugee** [comment]
I just realized I never followed up on this, sorry! The long-term fix which I've been working on is #42. Hopefully this will land soon; see https://github.com/strugee/node-crawl-mf2
